src/sentry/static/sentry/app/components/versionHoverCard.jsx

```diff
@@ -116,75 +134,123 @@ const VersionHoverCard = React.createClass({
   },
 
   renderBody() {
-    let {release} = this.state;
+    let {release, deploys} = this.state;
     let {version} = this.props;
-
-    let lastCommit = release.lastCommit;
+    let lastCommit = release && release.lastCommit;
```

Comment:
is there actually a chance `release` will be null? because if so, it's going to cause problems in line 164 anyway